src/components/common/vatsim/CommonEventCard.vue

```diff
@@ -0,0 +1,181 @@
+<template>
+    <div>
+        <div
+            class="event-card"
+            :class="{ 'active-event': active }"
+            @click="details = !details"
+        >
+            <div class="event-card_start">
+                {{ formattedStart }}Z - {{ formattedEnd }}Z
+            </div>
+            <div class="event-card_start">
+                {{ props.event.organisers[0]?.region }} -> {{ props.event.organisers[0]?.division }}
```

Comment:
Since this is needed for template, let's add a global v-if to prevent this

![image](https://github.com/user-attachments/assets/f1262f82-6fd8-4ee5-ad5b-fb93b9014e78)

```suggestion
              <template v-if="props.event.organisers?.length">
                {{ props.event.organisers[0]?.region }} -> {{ props.event.organisers[0]?.division }}
              </template>
```